src/Interface.js

```diff
@@ -0,0 +1,248 @@
+import * as Constant from './Constant.js'
```

Comment:
This assumes a flat bundle. That is we might need to figure out the story for reconcilers. I'm all for esm modules being first class as far as that is concerned, that is delegating this to build time instead of the current runtime semantics we have with `createFactory`.